Hyper/PlayerData/Player.cs

```diff
@@ -4,35 +4,35 @@
 using Common;
 using Common.Meshes;
 using Common.UserInput;
+using Hyper.PlayerData.InventorySystem;
 using OpenTK.Mathematics;
 using Physics.Collisions;
 using Physics.ContactCallbacks;
 using Physics.RayCasting;
 using Physics.TypingUtils;
-using Player.InventorySystem;
 
-namespace Player;
+namespace Hyper.PlayerData;
 
-public class Player : Humanoid, IRayCaster, IContactEventListener
+internal class Player : Humanoid, IRayCaster, IContactEventListener
 {
-    public Inventory Inventory;
-
+    public readonly Inventory Inventory;
+    
     private readonly RayEndpointMarker _rayEndpointMarker;
 
-    private readonly float _rayOffset = 3f; // arbitrary offset to make sure that the ray won't intersect with the player's own collidable
+    private const float RayOffset = 3f; // arbitrary offset to make sure that the ray won't intersect with the player's own collidable
 
     public float RayMaximumT => 20.0f;
 
     public System.Numerics.Vector3 RayDirection => Conversions.ToNumericsVector(ViewDirection);
 
     public System.Numerics.Vector3 RayOrigin => PhysicalCharacter.Pose.Position
-            + Conversions.ToNumericsVector(ViewDirection) * _rayOffset;
+            + Conversions.ToNumericsVector(ViewDirection) * RayOffset;
 
     public int RayId => 0;
 
     public Player(PhysicalCharacter physicalCharacter, Context context, int currentSphereId = 0) : base(physicalCharacter, currentSphereId)
     {
-        Inventory = new Inventory(context);
+        Inventory = new Inventory(context, true);
```

Comment:
can we use named arguments whenever a literal is passed to a method? It's easier to read

Reply:
I might forget about this because resharper does it for me but I'll try